treescript/src/treescript/merges.py

```diff
@@ -115,6 +115,28 @@ def touch_clobber_file(repo_path):
             "Merge old head via |hg debugsetparents {} {}| CLOSED TREE DONTBUILD a=release".format(tagged_from_rev, to_branch),
             repo_path=repo_path,
         )
+        # Preserve tags.
+        patch_file = os.path.join(os.path.dirname(repo_path), "patch_file")
+        tag_diff = await run_hg_command(config, "diff", "-r", to_branch, os.path.join(repo_path, ".hgtags"), "-U9", return_output=True, repo_path=repo_path)
```

Comment:
If you are not apply the patch (see below), the `-U0` probably makes more sense. `-U` specifies the number of lines of context to include; since we are only looking at additions, we don't need to generate context.